spring-kafka/src/main/java/org/springframework/kafka/core/LiKafkaProducerFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.core;
+
+import org.springframework.context.ApplicationEvent;
+
+/**
+ * @author Gary Russell
+ * @since 5.2
+ *
+ */
+public class LiKafkaProducerFactory extends DefaultKafkaProducerFactory {
```

Comment:
Gary, I think this is side effect of your other work.
Please, take it away from the PR.

Reply:
Oops; fixed.